8/debian9/8.0/docker-entrypoint.sh

```diff
@@ -16,7 +16,12 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-set -eo pipefail
+if [[ ! -z "${DEBUG_DOCKER_ENTERYPOINT}" ]]; then
```

Comment:
Please revert this.

Reply:
Could you kindly explain why we should revert this change?
I my point of view this debug option can help us easily find problems with updating for new versions. With out this options we should rebuild container to enable debug.

Reply:
Could you add a warning if the debug mode is enabled?

For example:

```
if [[ ! -z "${DEBUG_DOCKER_ENTERYPOINT}" ]]; then
	echo "!!! WARNING: DEBUG_DOCKER_ENTERYPOINT is enabled!"
	echo "!!! WARNING: Use only for debugging. Do not use in production!"
	set -x
else
...
```

Reply:
Waring is added